spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/leader/JdbcLockRegistryLeaderInitiatorTests.java

```diff
@@ -74,6 +74,7 @@ public void testDistributedLeaderElection() throws Exception {
 			LockRegistryLeaderInitiator initiator = new LockRegistryLeaderInitiator(
 					new JdbcLockRegistry(lockRepository),
 					new DefaultCandidate("foo", "bar"));
+			initiator.afterPropertiesSet();
```

Comment:
When you reinstate the ctor behavior in the `LockRegistryLeaderInitiator`, all these changes in these tests won't be necessary.